### PR TITLE
Command overhaul

### PR DESCRIPTION
**Pull request by moelsayed**
remove cluster command
merge upgrade and up commands
rename down command

https://github.com/rancher/rke/issues/54

### CODE REVIEW

File: cmd/config.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ func getHostConfig(reader *bufio.Reader, index int) (*v1.RKEConfigHost, error) {
 	}
 	host.User = sshUser
 
-	dockerSocketPath, err := getConfig(reader, fmt.Sprintf("Docker socker path on host (%s)", advertisedHostname), "/var/run/docker.sock")
+	dockerSocketPath, err := getConfig(reader, fmt.Sprintf("Docker docker path on host (%s)", advertisedHostname), "/var/run/docker.sock")
```
Comment (galal-hussein) [inline cmd/config.go]: socket not docker

File: docker/docker.go [hunk 2]
```diff
@@ -50,6 +59,12 @@ func DoRollingUpdateContainer(dClient *client.Client, imageCfg *container.Config
 		logrus.Infof("[%s] Container %s is not running on host [%s]", plane, containerName, hostname)
 		return nil
 	}
+	logrus.Debugf("[%s] Pulling Image on host [%s]", plane, hostname)
```
Comment (galal-hussein) [inline docker/docker.go]: can this be info not debug

File: cmd/remove.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+	"github.com/rancher/rke/cluster"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+func RemoveCommand() cli.Command {
+	removeFlags := []cli.Flag{
+		cli.StringFlag{
+			Name:   "cluster-file",
```
Comment (alena1108) [inline cmd/remove.go]: as we imply `cluster` in `rke up/remove`, should we rename this parameter to `config`?

Comment (alena1108) [reply cmd/remove.go]: It would also align with `generate config` command better

File: cmd/up.go [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+	"github.com/rancher/rke/pki"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+	"k8s.io/client-go/util/cert"
+)
+
+func UpCommand() cli.Command {
+	upFlags := []cli.Flag{
+		cli.StringFlag{
+			Name:   "cluster-file",
```
Comment (alena1108) [inline cmd/up.go]: Same here, we should consider renaming it.

File: cmd/version.go [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+		cli.StringFlag{
+			Name:   "cluster-file",
+			Usage:  "Specify an alternate cluster YAML file",
+			Value:  cluster.DefaultClusterConfig,
+			EnvVar: "CLUSTER_FILE",
+		},
+	}
+	return cli.Command{
+		Name:   "version",
+		Usage:  "Show Cluster Kubernetes version",
```
Comment (alena1108) [inline cmd/version.go]: In other commands descriptions, we have `cluster` lowercased, should probably do the same here

File: cmd/version.go [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+import (
+	"fmt"
+
+	"github.com/rancher/rke/cluster"
+	"github.com/urfave/cli"
+)
+
+func VersionCommand() cli.Command {
+	versionFlags := []cli.Flag{
+		cli.StringFlag{
```
Comment (alena1108) [inline cmd/version.go]: Rename the param

**Pull request merged**